[Merge-Queue] Use contributor triggering commit as committer

JonWBedard opened the pull request:
#### a43dac729e7e8d1cafa48e12575204305b964f54
<pre>
[Merge-Queue] Use contributor triggering commit as committer
<a href="https://bugs.webkit.org/show_bug.cgi?id=241923">https://bugs.webkit.org/show_bug.cgi?id=241923</a>
&lt;rdar://95793938&gt;

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(Contributors.load): Set emails for contributors.
(Canonicalize):
(Canonicalize.__init__): Cache contributors.
(Canonicalize.run): Set committer.
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: <a href="https://commits.webkit.org/251801@main">https://commits.webkit.org/251801@main</a>
</pre>

aj062 commented on Tools/CISupport/ews-build/steps.py:
```diff
@@ -493,8 +493,16 @@ def load(cls, use_network=None):
+                cls.contributors[name] = {
+                    'name': name,
+                    'status': value.get('status'),
+                    'email': bugzilla_email,
```
Nit: not sure if we should call it bugzilla email anymore or just email.

JonWBedard replied:
Good point, will change

aj062 reviewed (commented):
rs=me

JonWBedard committed:
[Merge-Queue] Use contributor triggering commit as committer
https://bugs.webkit.org/show_bug.cgi?id=241923
<rdar://95793938>

Reviewed by Aakash Jain.

* Tools/CISupport/ews-build/steps.py:
(Contributors.load): Set emails for contributors.
(Canonicalize):
(Canonicalize.__init__): Cache contributors.
(Canonicalize.run): Set committer.
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: https://commits.webkit.org/251801@main
